tools/js-sdk-release-tools/src/utils/getOutputPackageInfo.ts

```diff
@@ -16,8 +16,8 @@ export function getOutputPackageInfo(runningEnvironment: RunningEnvironment | un
             "artifacts": [],
             "result": "succeeded"
         };
-        if (cadlProject) {
-            outputPackageInfo['cadlProject'] = [cadlProject];
+        if (typeSpecProject) {
+            outputPackageInfo['typeSpecProject'] = [typeSpecProject];
```

Comment:
This is the naming convention used in other places: either `typespec` or `TypeSpec`. So, please follow it. I just made the changes to the contract in SDK automation framework.